src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/UtxoReconciliate/UtxoReconciliate.tsx

```diff
@@ -0,0 +1,258 @@
+        fiatAmount: '',
+        fiatSymbol: selectedCurrency,
+        vanityAddress: '',
+        input: accountAddress,
+      }
+
+      return sendInput
+    } catch (e) {
+      moduleLogger.error({ fn: 'getSendInput', e }, 'Error building THORChain savers Tx')
+    }
+  }, [accountId, asset, accountAddress, estimatedFees, query.cryptoAmount, selectedCurrency])
+
+  const handleBack = useCallback(() => {
+    history.push({
+      pathname: location.pathname,
+      search: qs.stringify({
+        ...query,
+        modal: DefiAction.Overview,
+      }),
+    })
+  }, [history, location.pathname, query])
+
+  const handleConfirm = useCallback(async () => {
+    setLoading(true)
+    const utxoReconciliateSendInput = getUtxoReconciliateSendInput()
+    const maybeTxId = await handleSend({
+      sendInput: utxoReconciliateSendInput!,
+      wallet: walletState.wallet!,
+    })
+
+    setTxId(maybeTxId)
+  }, [getUtxoReconciliateSendInput, walletState.wallet])
+
+  return (
+    <Flex
+      width='full'
+      minWidth={{ base: '100%', md: '500px' }}
+      maxWidth={{ base: 'full', md: '500px' }}
+      flexDir='column'
+    >
+      <DefiModalHeader
+        title={translate('defi.modals.saversVaults.utxoReconciliate')}
+        onBack={handleBack}
+      />
+      <Stack spacing={0} divider={<Divider />}>
+        <Row variant='vert-gutter' gap={2} alignItems='center'>
+          <Row.Label>
+            <Avatar
+              icon={<ArrowDownIcon />}
+              colorScheme='blue'
+              bg={useColorModeValue('gray.200', 'gray.700')}
+              color={useColorModeValue('blue.500', 'blue.200')}
+            >
+              <WarningIcon
+                position='absolute'
+                color={useColorModeValue('yellow.500', 'yellow.200')}
+                right={0}
+                top={0}
+              />
+            </Avatar>
+          </Row.Label>
+          <Row.Value textAlign='center'>
+            <Text
+              fontWeight='medium'
+              translation={`defi.modals.saversVaults.utxoReconciliateBody.${
+                query.reconciliationType === ReconciliationType.Deposit ? 'sendAmount' : 'sendDust'
+              }`}
+            />
+          </Row.Value>
+        </Row>
+        <Row variant='vert-gutter' alignItems='center' gap={0} py={8}>
+          <Row.Value>
+            <Amount.Crypto fontSize='3xl' value={query.cryptoAmount} symbol='BTC' />
+          </Row.Value>
+          <Row.Label>
+            <Amount.Fiat fontSize='2xl' value={query.fiatAmount} color='gray.500' prefix='≈' />
+          </Row.Label>
+        </Row>
+        <Summary bg='transparent' borderWidth={0} px={8} py={6} divider={<></>}>
+          <Row variant='gutter' px={0}>
+            <Row.Label>{translate('defi.modals.saversVaults.addressToSendTo')}</Row.Label>
+            <Row.Value fontSize='md'>
+              <Address
+                explorerAddressLink={`${asset?.explorerTxLink}/${txId}`}
```

Comment:
Don't want to clutter this diff with it, but extracting all `useColorModeValue` calls at JSX-level to variables will keep the order of hooks and allow us to `if (!asset) return null` instead of opt-chaining here